servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancerFactory.java

```diff
@@ -95,13 +121,89 @@ public Builder() {
             return this;
         }
 
+        /**
+         * This {@link LoadBalancer} may monitor hosts to which connection establishment has failed
+         * using health checks that run in the background. The health check tries to establish a new connection
+         * and if it succeeds, the host is returned to the load balancing pool. As long as the connection
+         * establishment fails, the host is not considered for opening new connections for processed requests.
+         * <p>
+         * {@link #healthCheckFailedConnectionsThreshold(int)} can be used to disable this mechanism and always
+         * consider all hosts for establishing new connections.
+         * </p>
+         *
+         * @param backgroundExecutor {@link Executor} on which to schedule health checking.
+         * @return {@code this}.
+         * @see #healthCheckFailedConnectionsThreshold(int)
+         */
+        public RoundRobinLoadBalancerFactory.Builder<ResolvedAddress, C> backgroundExecutor(
+                Executor backgroundExecutor) {
+            this.backgroundExecutor = requireNonNull(backgroundExecutor);
+            return this;
+        }
+
+        /**
+         * Configure an interval for health checking a host that failed to open connections.
+         * <p>
+         * {@link #healthCheckFailedConnectionsThreshold(int)} can be used to disable the health checking mechanism
+         * and always consider all hosts for establishing new connections.
+         * </p>
+         * @param interval interval at which a background health check will be scheduled.
```

Comment:
Should mention that providing an interval is optional and an uspecified default value will be used if not specified.